docs/dictionary/command/stop-session.lcdoc

```diff
@@ -20,6 +20,6 @@ Use the <stop session> command to end the session explicitly and save the conten
 
 The location where the $_SESSION array is stored is configured using the <sessionSavePath property>.
```

Comment:
To be clear, this refers to `<sessionSavePath property>` on line 21 - this should be `<sessionSavePath> property`, so that the text reads 
```
sessionSavePath property
```
but the link is only on the text `sessionSavePath`

As it is at the moment, `<sessionSavePath property>` will look for an entry called `sessionSavePath property`, not an entry called `sessionSavePath`.